src/backend/core/tests/teams/test_core_api_teams_create.py

```diff
@@ -48,3 +48,35 @@ def test_api_teams_create_authenticated():
     team = Team.objects.get()
     assert team.name == "my team"
     assert team.accesses.filter(role="owner", user=user).exists()
+
+
+def test_api_teams_create_authenticated_slugify_name():
+    """
+    Creating teams should automatically generate their slug.
+    """
+    identity = IdentityFactory()
+    jwt_token = OIDCToken.for_user(identity.user)
+
+    response = APIClient().post(
+        "/api/v1.0/teams/",
+        {
+            "name": "my team",
+        },
+        HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
+    )
+
+    assert response.status_code == 201
+    team = Team.objects.get()
+    assert team.name == "my team"
+    assert team.slug == "my-team"
+
+    response = APIClient().get(
+        "/api/v1.0/teams/my-team/",
+        {
+            "name": "my team",
+        },
+        HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
+    )
+
+    print(response.json())
+    assert response.status_code == 200
```

Comment:
EOF file, should we add a linter rule?

Reply:
There is one. My linter is now up and running aaagaïn !